### ISSUE TITLE
What's the matter? This always happens (run agent schedule 'every_1m') in Background Jobs

### ISSUE THREAD

**Issue by log6262635** [huginn/huginn#3067]
I don't use Delayagent, and Schedule is Never。What's more surprising is that this will happen even if I delete all agents

queued | Run Agent schedule 'every_1m' | 5m ago | 5m ago | 0 |   | Delete
queued | Run Agent schedule 'every_2m' | 5m ago | 5m ago | 0 |   | Delete
queued | Run Agent schedule 'every_5m' | 5m ago | 5m ago | 0 |   | Delete
queued | Run Agent schedule 'every_10m' | 5m ago | 5m ago | 0 |   | Delete
queued | Run Event propagation | 5m ago | 5m ago | 0 |   | Delete
queued | Run Agent schedule 'every_1m' | 4m ago | 4m ago | 0 |   | Delete
queued | Run Agent schedule 'every_1m' | 3m ago | 3m ago | 0 |   | Delete
queued | Run Agent schedule 'every_2m' | 3m ago | 3m ago | 0 |   | Delete
queued | Run Agent schedule 'every_1m' | 2m ago | 2m ago | 0 |   | Delete
queued | Run Agent schedule 'every_1m' | 1m ago | 1m ago | 0 |   | Delete
queued | Run Agent schedule 'every_2m' | 1m ago | 1m ago | 0 |   | Delete
queued | Run Agent schedule 'every_1m' | <1m ago | <1m ago | 0 |   | Delete
queued | Run Agent schedule 'every_5m' | <1m ago | <1m ago | 0 |   | Delete

**Comment by virtadpt** [comment]
That is the scheduler walking through the agent queues, looking for agents that need to be executed.  This is normal.

**Comment by gingerbeardman** [comment]
I also found this unexpected, since none of my agents execute at per-minute granularity (only hour granularity)

**Comment by ww2283** [comment]
I think this is not a normal behavior. I see a strong correlation between a Website Agent which failed to fetch its value but stands on the top of job list. It says "running" but doing actually nothing. In that case, the queued 1m jobs pile up. I tried to delete the agent but situation persisted. I had to do a restart of docker to get rid of the piled queue. Try the following setup and see if you can reproduce the problem:
{
  "expected_update_period_in_days": "2",
  "url": "https://www.bestbuy.com/site/nvidia-geforce-rtx-4090-24gb-gddr6x-graphics-card-titanium-and-black/6521430.p?skuId=6521430",
  "type": "html",
  "mode": "on_change",
  "extract": {
    "Price": {
      "xpath": "//*[@id="fulfillment-fulfillment-summary-11719805"]/div/div/div/div[2]/div[1]/strong",
      "value": "string(.)"
    }
  },
  "headers": {
    "user-agent": "Mozilla/5.0"
  }
}

**Comment by gingerbeardman** [comment]
@ww2283 that seems to be a different issue. whenever I get a "running" stuck at the top of queue, I reboot Huginn as that state seems unrecoverable. but I don't think it's the same as this issue.

The behaviour described in this thread is Huggin checking every minute regardless of the schedule of agents. Some minutes it will find agents to execute, others it will not.

**Comment by ww2283** [comment]
@gingerbeardman yes I think you are right. Besides it is more of a problem from bestbuy I guess. Nothing from bestbuy can be parsed with Website Agent. Dry run will stay silent as well. I think that's why it is stuck.
